src/riak_ql_parser.yrl

```diff
@@ -931,6 +950,8 @@ get_func_type(FuncName) when FuncName =:= 'AVG'    orelse
                              FuncName =:= 'STDDEV_SAMP' orelse
                              FuncName =:= 'STDDEV_POP' ->
     window_aggregate_fn;
+get_func_type('TIME') ->
+    sql_select_fn;
```

Comment:
The atom name could be more expressive. As is, `sql_select_fn` just tells me it is a function used in the SELECT clause, but so also are the window aggregate functions. I would suggest `time_quantum_fn` for it instead.